scripts/gen_graphs.py

```diff
@@ -101,6 +103,31 @@ def generate_time_xy_issue_graph(oss_entity,data_key):
     write_repo_chart_to_file(oss_entity, xy_time_issue_chart, data_key)
 
 
+def generate_donut_graph_line_complexity_graph(oss_entity):
+    """
+    This function generates pygals line complexitydonut graph
+    for a set of Repository objects.
+
+    Arguments:
+        oss_entity: The OSSEntity to create a graph for.
```

Comment:
IDEA: What is an OSSEntity? Perhaps we could say what the value typically is, where it comes from, what it is used for?

Reply:
I added something about what is is typically to the docstring